adb_graphics/datahandler/gribdata.py

```diff
@@ -526,7 +533,7 @@ def corners(self) -> list:
         '''
 
         lat, lon = self.latlons()
-        if self.model in ['global', 'obs']:
+        if self.model in ['global', 'obs'] and self.model != 'globalAK':
```

Comment:
The original logic only looks for `self.model == 'global'` and `self.model == 'obs'`, so I'm not sure this additional logic is needed for `globalAK`.

Reply:
I'm not sure why I thought this was needed.  I've removed it.